### PR TITLE
Run tests with python 3.12

### PR DESCRIPTION
**Pull request by jairhenrique**

### CODE REVIEW

**Comment by jairhenrique** [comment]
https://github.com/globocom/derpconf/issues/29

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -21,33 +37,42 @@ jobs:
...
+          python-version: ${{ matrix.image-tag }}
+      - name: Install SO dependencies
+        run: sudo apt-get update && sudo apt-get install -y libwebp-dev libjpeg-dev libssl-dev zlib1g-dev ffmpeg scons gifsicle libcurl4-openssl-dev libgnutls28-dev libjpeg-progs libimage-exiftool-perl
+      # - name: Build test image
+      #   run: docker build -t test_image -f TestDockerfile --build-arg PYTHON_VERSION=${{ matrix.image-tag }} .
+      # - name: Start test container
+      #   run: |
+      #     docker run -dt --name thumbor_test -v $(pwd):/app test_image
+      # - name: Fire up Redis
+      #   # run: docker exec thumbor_test make redis
```
Comment (RaphaelVRossi) [inline .github/workflows/build.yml:60]: Hey @jairhenrique,

Thumbor needs redis single node and sentinel instance running to test itself.

Maybe another service with sentinel configs should work.

**Comment by guilhermef** [comment]
@jairhenrique, there's an issue with the newest Pillow release: #1608

File: .github/workflows/build.yml (hunk 1 above)
Comment (jairhenrique) [reply .github/workflows/build.yml]: I remove this commit, it's only a test.

**Comment by jairhenrique** [comment]
Close in favor of #1608

**Pull request closed without merging**